src/libs/CardFeedUtils.ts

```diff
@@ -446,6 +446,41 @@ function getCardFeedsForDisplay(allCardFeeds: OnyxCollection<CardFeeds>, allCard
     return cardFeedsForDisplay;
 }
 
+/**
+ * Given a collection of card feeds, return formatted card feeds grouped per policy.
+ *
+ * Note: "Expensify Card" feeds are not included.
+ */
+function getCardFeedsForDisplayPerPolicy(allCardFeeds: OnyxCollection<CardFeeds>): Record<string, CardFeedForDisplay[]> {
+    const cardFeedsForDisplayPerPolicy = {} as Record<string, CardFeedForDisplay[]>;
+
+    Object.entries(allCardFeeds ?? {}).forEach(([domainKey, cardFeeds]) => {
+        // sharedNVP_private_domain_member_123456 -> 123456
+        const fundID = domainKey.split('_').at(-1);
+        if (!fundID) {
+            return;
+        }
+
+        Object.entries(getCompanyFeeds(cardFeeds, true, true)).forEach(([key, feedData]) => {
+            if (!('preferredPolicy' in feedData) || !feedData.preferredPolicy) {
+                return;
+            }
```

Comment:
The first condition is required because `DirectCardFeedData` does not include `preferredPolicy`. Is that intended? Aren't direct card feeds linked to policies?